templates/core/unauthorized.html.twig

```diff
@@ -0,0 +1,7 @@
+<div class="d-flex w-100 h-75 align-items-center justify-content-center">
+    <h3>
+        {{ pageTitle|xlt ~ " Not Authorized"|xlt }}
```

Comment:
as discussed previously, better to translate constant where it is known than in a variable. So would consider translating pageTitle previously and then just running it through text here.
Also, for constants, no leading or trailing space in the constants.
So prob would look something like:
```
{{ pageTitle|text }} {{ "Not Authorized"|xlt }}
```